chia/_tests/core/mempool/test_mempool_manager.py

```diff
@@ -245,7 +246,7 @@ class TestCheckTimeLocks:
         coinbase=False,
         timestamp=COIN_TIMESTAMP,
     )
-    REMOVALS: dict[bytes32, CoinRecord] = {TEST_COIN.name(): COIN_RECORD}
+    REMOVALS: dict[bytes32, CoinRecord] = dataclasses.field(default_factory=lambda: {TEST_COIN.name(): COIN_RECORD})  # type: ignore[name-defined]  # noqa: F821
```

Comment:
Again, this is me addressing a concern about the mutable default.  The linters still don't like it because they don't recognize `COIN_RECORD` as a valid variable, but it definitely passes the tests 🤷

Reply:
taken